packages/ra-data-simple-prisma/src/extractWhere.ts

```diff
@@ -5,8 +5,10 @@ import setObjectProp from "set-value";
 
 const logicalOperators = ["gte", "lte", "lt", "gt"];
 
+export type FilterMode = "insensitive" | "default" | undefined;
+
 type ExtractWhereOptions = {
-  mode?: "insensitive" | "default";
+  filterMode?: FilterMode;
```

Comment:
@heba-khazbak sorry but this was causing issues with people using DBs not supporting insensitive mode https://www.prisma.io/docs/concepts/components/prisma-client/case-sensitivity#options-for-case-insensitive-filtering

Reply:
Will need to be explicitly passed as an option to getManyReference or getList